.gitignore

```diff
@@ -13,6 +13,8 @@ dependency-reduced-pom.xml
 
 # gradle

```

Comment:
Don't forget to remove maven stuff

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/51/commits/09ed7d5abb92be92f58d1c0c2360816bbb756195